OctoKit/Event.swift

```diff
@@ -0,0 +1,147 @@
+import Foundation
+import RequestKit
+
+public extension Octokit {
+
+    /**
+     Fetches all recent Events on the server
+     - parameter session: RequestKitURLSession, defaults to NSURLSession.sharedSession()
+     - parameter completion: Callback for the outcome of the fetch.
+     */
+    @discardableResult
+    func allEvents(_ session: RequestKitURLSession = URLSession.shared,
+                  completion: @escaping (_ response: Response<[Event]>) -> Void) -> URLSessionDataTaskProtocol? {
+
+        let router = EventRouter.allEvents(self.configuration)
+        return router.load(session, expectedResultType: [Event].self) { events, error in
+            if let error = error {
+                completion(Response.failure(error))
+            } else {
+                if let events = events {
+                    completion(Response.success(events))
+                }
+            }
+        }
+    }
+
+    /**
+     Fetches all Received Events
```

Comment:
Same here link to githbub docs.

Get more pages by reading headers, 
expand API to allow count or date?